components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -6401,22 +6673,29 @@ def reexportedName(name):
                 cgThings.append(CGProxyUnwrap(descriptor))
                 cgThings.append(CGDOMJSProxyHandlerDOMClass(descriptor))
                 cgThings.append(CGDOMJSProxyHandler_ownPropertyKeys(descriptor))
-                if descriptor.interface.getExtendedAttribute("LegacyUnenumerableNamedProperties"):
+                if descriptor.interface.getExtendedAttribute("LegacyUnenumerableNamedProperties") or \
+                   descriptor.isMaybeCrossOriginObject():
                     cgThings.append(CGDOMJSProxyHandler_getOwnEnumerablePropertyKeys(descriptor))
                 cgThings.append(CGDOMJSProxyHandler_getOwnPropertyDescriptor(descriptor))
                 cgThings.append(CGDOMJSProxyHandler_className(descriptor))
                 cgThings.append(CGDOMJSProxyHandler_get(descriptor))
                 cgThings.append(CGDOMJSProxyHandler_hasOwn(descriptor))
 
-                if descriptor.operations['IndexedSetter'] or descriptor.operations['NamedSetter']:
+                if descriptor.isMaybeCrossOriginObject() or descriptor.operations['IndexedSetter'] or \
+                   descriptor.operations['NamedSetter']:
                     cgThings.append(CGDOMJSProxyHandler_defineProperty(descriptor))
 
                 # We want to prevent indexed deleters from compiling at all.
                 assert not descriptor.operations['IndexedDeleter']
 
-                if descriptor.operations['NamedDeleter']:
+                if descriptor.isMaybeCrossOriginObject() or descriptor.operations['NamedDeleter']:
                     cgThings.append(CGDOMJSProxyHandler_delete(descriptor))
 
+                if descriptor.isMaybeCrossOriginObject():
+                    cgThings.append(CGDOMJSProxyHandler_getPrototype(descriptor))
+                    cgThings.append(CGDOMJSProxyHandler_set(descriptor))
+                    pass
```

Comment:
nit: remove the `pass`.

Reply:
Addressed in 690d8462a5105f4662c3c0bc6ab0c0d8c414b040